osu.Game/Screens/Play/HUD/SongProgress.cs

```diff
@@ -14,6 +14,12 @@ namespace osu.Game.Screens.Play.HUD
 {
     public abstract class SongProgress : OverlayContainer, ISkinnableDrawable
     {
+        // Some implementations of this element allow seeking during gameplay playback.
+        // Set a sane default of never handling input to override the behaviour provided by OverlayContainer.
+        public override bool HandleNonPositionalInput => false;
+        public override bool HandlePositionalInput => false;
```

Comment:
I don't know how this works, but it seems to work.

Reply:
Yeah, rather than over-thinking I just copied what is already working for `DefaultSongProgress`.

Reply:
Basically `OverlayContainer` steals mouse/key/hover events from the gameplay cursor/input layer (which in turn switches the cursor from gameplay to menu, because of the [`HoveredDrawables`-based cursor visibility logic](https://github.com/ppy/osu/blob/ef10145d6f9f77117e6e148cf82f76f6b89ffdb4/osu.Game/Graphics/Cursor/GlobalCursorDisplay.cs#L74-L81)), so when overriding these to `false`, the container no longer steals events and input is received normally.